docs/get-details/transactions/index.md

```diff
@@ -675,19 +675,56 @@ Here are three example scenarios and how the round range may be calculated for e
 
 # Fees
 
-There are two primary ways to set the fee for a transaction. 
+Fees for transactions on Algorand are set as a function of network congestion and based on the size in bytes of the transaction.  Every transaction must at least cover the minimum fee (1000µA or 0.001A). 
 
-## Suggested Fee
+For a transaction serialized to bytes (`txn_in_bytes`) and current congestion based fee per byte (`fee_per_byte`) the fee can be computed as follows:
+
+```py
+fee = max(current_fee_per_byte*len(txn_in_bytes), min_fee)
+```
+
+If the network *is not* congested, the fee per byte will be 0 and the minimum fee for a transaction is used. 
+
+If network *is* congested the fee per byte will be non zero and for a given transaction will be the product of the size in bytes of the transaction and the current fee per byte. If the product is less than the min fee, the min fee is used. 
 
```

Comment:
```suggestion

Note that fees are independent of the type of transaction (payment, ASA transfer, application call, ...) and a fortiori independent of the complexity of the smart contract code in case of application calls.
Only the size of the serialized transaction matters.

```